notes/v1.0.0-beta1.md

```diff
@@ -152,131 +148,67 @@ The command now works as expected.
 <!-- AUTOGENERATED COMMIT LIST -->
 
 97c3985 Adding ecctl icon to repo (#111)
```

Comment:
The changes look like this in markdown, are we OK without the newlines?

![Screen Shot 2020-01-03 at 10 07 05 AM](https://user-images.githubusercontent.com/7670244/71693294-f8fe6f00-2e10-11ea-82b2-a2d751bec913.png)

Reply:
That's very strange. My local parser returns it as it's been published in the release, and also the release notes has been published with the same format as this PR publishes. Not sure why GitHub preview returns it as we see it.

Reply:
Looks like it's always shown like this in the [previous releases](https://github.com/elastic/cloud-cli/blob/master/notes/1.0.0-bc1.md#detailed-changes) 🤔 Do we want to keep it like this?